apps/omg_watcher/lib/omg_watcher/api/transaction.ex

```diff
@@ -32,7 +32,7 @@ defmodule OMG.Watcher.API.Transaction do
   @doc """
   Retrieves a specific transaction by id
   """
-  @decorate measure_event()
+
```

Comment:
Extra line? (I will stop commenting on these now, sorry)

Reply:
It appears like there's an extra line, but it's actually not. I think it has to do with GH web formatting of diffs?